test/performance/Makefile.am

```diff
@@ -11,7 +11,8 @@
 
 check_PROGRAMS = $(TESTS)
 
-TESTS =
+TESTS = \
+	shmem_put_perf
```

Comment:
It also need to be added to ```check_PROGRAMS``` so that we always compile the test case, even when we don't run it as part of ```make check```.  Part of the bit rot prevention program.  :)